README.md

```diff
@@ -1 +1,135 @@
-# Hello-Microverse
\ No newline at end of file
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 [Hello Microverse project] <a name="about-project"></a>
+
+**[Hello Microverse project]** is a project which is done in the partial fulfillment of concepts like git,github, css and html lint.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+  
+</details>
+
+<details>
+  <summary>CSS</summary>
+  
+</details>
+
+<details>
+<summary>Linters</summary>
+  <ul>
+    <li>Lighthouse</li>
+    <li>Webhint</li>
+    <li>Stylelint</li>
+  </ul>
+</details>
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://github.com/kifle23/Hello-Microverse)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+<ul dir="auto">
+<li>Installation of Git</li>
+<li>Installation of NPM</li>
+<li>A working browser</li>
+<li>Internet connection</li>
+<li>A code editor like Visual Studio Code</li>
+<li>Installation of Live Server plugging in Visual Studio Code</li>
+</ul>
+
+
+### Prerequisites
+ 
+ nodejs 
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  -cd my-folder
+  -git clone git@github.com:kifle23/Hello-microverse.git
+
+### Install
+
+Install this project with:
+
+-[coming soon]
+
+### Usage
+
+To run the project, execute the following command:
+
+-[coming soon]
+
+### Run tests
+
+To run tests, run the following command:
+-[coming soon]
+
+
+### Deployment
+
+You can deploy this project using:
+
+-[coming soon]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Kifle Haile**
+
+- GitHub: [@githubhandle](https://github.com/kifle23/)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/kifle-haile-5a613761/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/kifle23/Hello-Microverse/issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give it some ⭐️s
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] Great job on adding missing sections but according to the content table there are still two of the sections missing. Please make sure to include `future features` section inlist 2-3 future features of the project and `Acknowlegment` section to give credit to the people who's content has been used in the project